components/script/dom/document.rs

```diff
@@ -2889,7 +2891,7 @@ impl DocumentMethods for Document {
 
     // https://dom.spec.whatwg.org/#dom-document-content_type
     fn ContentType(&self) -> DOMString {
-        self.content_type.clone()
+        DOMString::from_string(self.content_type.0.to_string() + "/" + &self.content_type.1.to_string()).clone()
```

Comment:
We can do `DOMString::from(self.content_type.as_str())`. We can remove the `.clone()` now, too.

Reply:
I recently found that .as_str() method doesn't actually exist for Mime type, is there any other method other than the one I used, which will look better here?